projects/laji-ui/src/lib/_vars.scss

```diff
@@ -9,6 +9,7 @@ $sp-offset: 0px !default;
 */
 
 $breakpoints: (
+  "xl": 1600px,
```

Comment:
Could this use `$xl-width` etc.?

Reply:
For every other var (sm, md, lg) those numbers are different between breakpoints and the widths